archivist/archivistpublic.py

```diff
@@ -151,6 +157,16 @@ def max_time(self) -> float:
         """bool: Returns maximum time in seconds to wait for confirmation"""
         return self._max_time
 
+    @property
+    def version(self) -> str:
+        """str: Returns version of the archivist package"""
+        return VERSION
+
+    @property
+    def partner_id(self) -> str:
+        """str: Returns partner id of the archivist package"""
```

Comment:
This looks wrong: surely it's the partner ID the package was initialised with, not some static release property?

Reply:
fixed